### PR TITLE
Updating links for KubeCon China

### PR DESCRIPTION
**Pull request by kbarnard10**
Updating links for KubeCon China on homepage, blog, and community page.

### CODE REVIEW

File: content/en/blog/_posts/2018-06-26-kubernetes-1.11-release-announcement.md [hunk 1]
```diff
@@ -92,7 +92,7 @@ Is Kubernetes helping your team? Share your story with the community.
...
+The world’s largest Kubernetes gathering, KubeCon + CloudNativeCon is coming to [Shanghai](https://events.linuxfoundation.cn/events/kubecon-cloudnativecon-china-2018/ from November 14-15, 2018 and [Seattle](https://events.linuxfoundation.org/events/kubecon-cloudnativecon-north-america-2018/) from December 11-13, 2018. This conference will feature technical sessions, case studies, developer deep dives, salons and more! The CFP for both event is currently open. [Submit your talk](https://events.linuxfoundation.org/events/kubecon-cloudnativecon-north-america-2018/program/call-for-proposals-cfp/) and [register](https://events.linuxfoundation.org/events/kubecon-cloudnativecon-europe-2018/attend/register/) today!
```
Comment (xiangpengzhao) [inline content/en/blog/_posts/2018-06-26-kubernetes-1.11-release-announcement.md]: Seems like the link of `[register]` is incorrect :)

**Comment by zacharysarah** [comment]
@kbarnard10 Whoops, this fell off the radar--looks like the `/lgtm` dropped with a new commit. LGTMing now, feel free to `/hold cancel` when ready. 

/lgtm
/hold

**Pull request merged**